conf/application.conf

```diff
@@ -75,14 +74,6 @@ microservice {
   }
 }
 
-metrics {
```

Comment:
May I know, why metrics and auditing is removed here?

Reply:
As discussed in our call earlier, this is to disable metrics and auditing locally